extensions/big-data-cluster/src/kubectl/host.ts

```diff
@@ -0,0 +1,42 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+
+export interface Host {
```

Comment:
In most of our other extensions we've written an ApiWrapper class that handles all of this. Could we use this instead? Or is there some mapping going on that benefits from doing it this way?

Reply:
For now I moved over the design from the vscode kubernetes extension to move fast. I can work on refactoring to a APIWrapper after I get the core scenario up and running.

Reply:
Sounds good. No pressure if keeping this makes doing future syncs easier.